docs/contributtion/experimental-code.md

```diff
@@ -0,0 +1,167 @@
+individual entries for each unstable export. These contain stories and written
+documentation outlining the functionality of these components and exports.
+
+## Feature flags
+
+Some Spirit packages ship with available feature flags. These feature flags
+enable new behavior and styling, allowing you to opt-in to new breaking changes
+while remaining on the current major version. When a new feature flag is
+introduced it is marked `false` or "off" by default to ensure backwards
+compatibility. A feature flag may be configured in JavaScript, SASS, or both.
+
+All the currently available feature flags, as well as how to enable them, is
+documented in the each package main README file in the section `Feature Flags`.
+
+### Feature flag naming convention
+
+All feature flags follow a prefix naming convention that indicate status.
+
+#### Flags prefixed with `enable-*`
+
+- Contain new features that we'd like consuming projects to test
+- Are generally stable and unlikely to change but may change based on user
+  feedback
+- May require some manual migration or code changes within your project
+- Are documented in the main package README file
+- Need user feedback to ensure we've met all concerns relating to this feature
+
+If you use these flags, make sure to check our release notes where we'll outline
+any changes to them across our releases.
+
+#### Flags prefixed with `enable-v#-*`
+
+As usage of an existing flag increases or we determine a feature to be of high
+importance, we'll "commit" it to a future major release and rename it to use the
+`enable-v#-*` prefix. At this point the API or functionality behind this flag is
+now fixed and won't change. We intend to ship this flag as "on by default" in
+the major version indicated in the name. e.g. `enable-v3-some-feature`
+
+All breaking changes will be shipped as `enable-v3-*` flags within the current
+major release (v3). This enables projects to opt-in to breaking changes earlier
+and at their own pace avoiding one huge changeset when upgrading to the next
+major release. In theory, if all `enable-v3-*` flags are enabled within your
+project before the v3 release, no changes should need to be made when updating
+to v3.
+
+For a flag to be committed to a release and renamed to `enable-v#-*` it must:
+
+- Be tested with early adopters
+- Be fully covered in tests (Unit, Visual)
+- Be documented in storybook
+- Be documented on https://spirit.design
+- Have an automated migration script (codemod) available, where possible
+
+### Documenting Feature Flags
+
+Within the experimental section of the `@lmc-eu/spirit-web-twig` Storybook, there is a
+"Feature Flags" folder. This contains stories and written documentation covering
+every available flag and how to configure them. Folders and stories within this
+section in storybook show components with all feature flags turned on.
+
+All the currently available feature flags, as well as how to enable them, is
+documented in the main package README file.
```

Comment:
Really? Currently we documented them only in the web package (we didn't have them anywhere else). And I can imagine that our users won't even know what do we have in our main package README.

Reply:
So what do you suggest? There should be a place where the feature flags of the package should be documented and I think that the main README of the package is the best place.

Reply:
Ok.